test/runtests.jl

```diff
@@ -85,12 +85,16 @@ ZMQ.close(ctx2)
 # deprecate bytestring(::Message)
 let olderr = STDERR
    rderr, wrerr = redirect_stderr()
-   @assert bytestring(Message("hello")) == "hello"
-   redirect_stderr(olderr)
+   try
+       reader = @async readstring(rderr)
```

Comment:
Hello,
to define done Task before the try
```
reader = ⓐasync nothing
try
    reader = ...
```

or, I prefer to remove `try finally` block that seems to work.
```diff
diff --git a/test/runtests.jl b/test/runtests.jl
index aaeb261..07dae84 100644
--- a/test/runtests.jl
+++ b/test/runtests.jl
@@ -85,15 +85,12 @@ ZMQ.close(ctx2)
 # deprecate bytestring(::Message)
 let olderr = STDERR
    rderr, wrerr = redirect_stderr()
-   try
-       reader = @async readstring(rderr)
-       @assert bytestring(Message("hello")) == "hello"
-   finally
-       redirect_stderr(olderr)
-   end
+   reader = @async readstring(rderr)
+   @assert bytestring(Message("hello")) == "hello"
+   redirect_stderr(olderr)
    close(wrerr)
    if VERSION < v"0.5-dev+4341"
-       @assert !contains(wait(reader), "WARNING: bytestring(zmsg::Message) is deprecated")
+       @assert isempty(wait(reader))
    else
        @assert contains(wait(reader), "WARNING: bytestring(zmsg::Message) is deprecated")
    end
```